src/contracts/core/DelegationManager.sol

```diff
@@ -625,6 +636,11 @@ contract DelegationManager is
 
             // Remove delegated shares from the operator
             if (operator != address(0)) {
+                // Staker was delegated and remains slashable during the withdrawal delay period
+                // Cumulative withdrawn scaled shares are updated for the strategy, this is for accounting
+                // purposes for burning shares if slashed
+                _updateCumulativeScaledShares(operator, strategies[i], scaledShares[i]);
```

Comment:
we don't want to do this for beaconchain ETH, because this value is sorta useless since it doesn't take slashing factors into account

Reply:
Good catch, yes this only for SM Strategies

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/b0ed123a0abfbec7c6f961097c74a850d708512d